<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaraSC1ConfigWidget</class>
 <widget class="QWidget" name="ScaraSC1ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>1528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="scara_sc1_config">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>SCARA SC1 Config</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="topMargin">
       <number>20</number>
      </property>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QGroupBox" name="program_settings">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>Program settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="topMargin">
              <number>10</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_16">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Main program file:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Main points file:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Sync freq:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <widget class="QComboBox" name="mainProgramComboBox">
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="mainPointsFileComboBox">
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="syncFreqLineEdit">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="xy_surface">
          <property name="title">
           <string>XY Surface</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>20</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Length of first segment ( L1 ) in mm:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lengthOfFirstSegmentLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Length of second segment ( L2 ) in mm</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lengthOfSecondSegmentLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Correction value ( C ) in mm:</string>
                   </property>
                   <property name="buddy">
                    <cstring>label_3</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="correctionValueLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>First segment limit angle on CW ( α ) in deg:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="firstSegmentLimitAngleOnCWLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>First segment limit angle on CCW ( β ) in deg:</string>
                   </property>
                   <property name="buddy">
                    <cstring>label_5</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="firstSegmentLimitAngleOnCCWLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Number of steps for first segment between CW and CCW:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>( This is necessary only for simulation )</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="numberOfStepsForFirstSegmentLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Second segment limit angle on CW ( γ ) in deg:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="secondSegmentLimitAngleOnCWLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Second segment limit angle on CCW ( δ ) in deg:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="secondSegmentLimitAngleOnCCWLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Number of steps for second segment between CW and CCW:</string>
                   </property>
                   <property name="buddy">
                    <cstring>numberOfStepsForSecondSegmentLineEdit</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>( This is necessary only for simulation )</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="numberOfStepsForSecondSegmentLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_20">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="firstSegmentBeginOnCCWRadioButton">
                   <property name="text">
                    <string>First segment begin on CCW</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="firstSegmentBeginOnCWRadioButton">
                   <property name="text">
                    <string>First segment begin on CW</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_18">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="secondSegmentBeginOnCCWRadioButton">
                   <property name="text">
                    <string>Second segment begin on CCW</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="secondSegmentBeginOnCWRadioButton">
                   <property name="text">
                    <string>Second segment begin on CW</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGraphicsView" name="scaraXYSurfaceGraphicsView">
               <property name="minimumSize">
                <size>
                 <width>550</width>
                 <height>614</height>
                </size>
               </property>
               <property name="midLineWidth">
                <number>-2</number>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
          <zorder></zorder>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="z_surface">
          <property name="title">
           <string>Z Surface</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="topMargin">
            <number>20</number>
           </property>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <property name="spacing">
                <number>20</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>Distance between two limits ( a ) in mm:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="distanceBetweenLimintsOnZLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>Length of third segment ( b ) in mm:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lengthOfThirdSegmentLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_13">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>Number of steps between two limits:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string>( This is necessary only for simulation )</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="numberOfStepsBetweenLimitsOnZLineEdit">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_22">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="increasingZonCWRadioButton">
                   <property name="text">
                    <string>Increasing Z on CW</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="increasingZonCCWRadioButton">
                   <property name="text">
                    <string>Increasing Z on CCW</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_19">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="thirdSegmentBeginOnCWRadioButton">
                   <property name="text">
                    <string>Begin on CW</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="thirdSegmentBeginOnCCWRadioButton">
                   <property name="text">
                    <string>Begin on CCW</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGraphicsView" name="scaraZSurfaceGraphicsView">
               <property name="minimumSize">
                <size>
                 <width>550</width>
                 <height>515</height>
                </size>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mainProgramComboBox</tabstop>
  <tabstop>mainPointsFileComboBox</tabstop>
  <tabstop>syncFreqLineEdit</tabstop>
  <tabstop>lengthOfFirstSegmentLineEdit</tabstop>
  <tabstop>lengthOfSecondSegmentLineEdit</tabstop>
  <tabstop>correctionValueLineEdit</tabstop>
  <tabstop>firstSegmentLimitAngleOnCWLineEdit</tabstop>
  <tabstop>firstSegmentLimitAngleOnCCWLineEdit</tabstop>
  <tabstop>numberOfStepsForFirstSegmentLineEdit</tabstop>
  <tabstop>secondSegmentLimitAngleOnCWLineEdit</tabstop>
  <tabstop>secondSegmentLimitAngleOnCCWLineEdit</tabstop>
  <tabstop>numberOfStepsForSecondSegmentLineEdit</tabstop>
  <tabstop>firstSegmentBeginOnCCWRadioButton</tabstop>
  <tabstop>firstSegmentBeginOnCWRadioButton</tabstop>
  <tabstop>secondSegmentBeginOnCCWRadioButton</tabstop>
  <tabstop>secondSegmentBeginOnCWRadioButton</tabstop>
  <tabstop>distanceBetweenLimintsOnZLineEdit</tabstop>
  <tabstop>lengthOfThirdSegmentLineEdit</tabstop>
  <tabstop>numberOfStepsBetweenLimitsOnZLineEdit</tabstop>
  <tabstop>increasingZonCWRadioButton</tabstop>
  <tabstop>increasingZonCCWRadioButton</tabstop>
  <tabstop>thirdSegmentBeginOnCWRadioButton</tabstop>
  <tabstop>thirdSegmentBeginOnCCWRadioButton</tabstop>
  <tabstop>scaraXYSurfaceGraphicsView</tabstop>
  <tabstop>scaraZSurfaceGraphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
